javascript/builtins/Promise.json

```diff
@@ -2,495 +2,493 @@
   "javascript": {
     "builtins": {
       "Promise": {
-        "Promise": {
```

Comment:
diff seems too many lines, but only what I did is to remove this L5.

Reply:
It’s best to view this locally in [kdiff3](https://chocolatey.org/packages/kdiff3).